modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/IncidentContactCard/IncidentContactCard.jsx

```diff
@@ -136,155 +139,171 @@ const IncidentContactCard = ({accountSubscriptionGroupsNames}) => {
-																)}
-															</p>
-														</>
-													)}
-
-													<br></br>
-												</div>
-											)
-										)}
+										{criticalIncidentContacts}
 									</div>
 								</div>
-							</>
-						)}
+
+								{isLXCEnvironment && (
+									<>
+										<div className="col customer-portal-card-description pl-4">
+											<h3>
+												{i18n.translate(
+													'security-breach'
+												)}
+
+												<ClayIcon symbol="pencil" />
+											</h3>
+
+											<div
+												className={`${
+													currentHighPriorityContacts
+														.privacyBreach?.length >
+													incidentContactStandard
+														? 'customer-portal-card-description-scroll scroller'
+														: ''
+												}`}
+											>
+												{currentHighPriorityContacts?.privacyBreach?.map(
+													({
+														contact,
+														email,
+														index,
+														label,
+													}) => (
+														<div key={index}>
+															<h4>{email}</h4>
+
+															<h5>{label}</h5>
+
+															{contact.length ? (
+																<h5>
+																	{contact}
+																</h5>
+															) : (
+																<>
+																	<p className="text-warning">
+																		<ClayIcon symbol="warning-full" />
+																		&nbsp;
+																		{i18n.translate(
+																			'phone-number-is-missing'
+																		)}
+																	</p>
+																</>
+															)}
+
+															<br></br>
+														</div>
+													)
+												)}
+											</div>
+										</div>
+
+										<div className="col customer-portal-card-description pl-4">
+											<h3>
+												{i18n.translate(
+													'privacy-breach'
+												)}
+
+												<ClayIcon symbol="pencil" />
+											</h3>
+
+											<div
+												className={`${
+													currentHighPriorityContacts
+														.securityBreach
+														?.length >
+													incidentContactStandard
+														? 'customer-portal-card-description-scroll scroller'
+														: ''
+												}`}
+											>
+												{currentHighPriorityContacts?.securityBreach?.map(
+													({
+														contact,
+														email,
+														index,
+														label,
+													}) => (
+														<div key={index}>
+															<h4>{email}</h4>
+
+															<h5>{label}</h5>
+
+															{contact.length ? (
+																<h5>
+																	{contact}
+																</h5>
+															) : (
+																<>
+																	<p className="text-warning">
+																		<ClayIcon symbol="warning-full" />
+																		&nbsp;
+																		{i18n.translate(
+																			'phone-number-is-missing'
+																		)}
+																	</p>
+																</>
+															)}
+
+															<br></br>
```

Comment:
Can we replace this br for margin bottom?

Reply:
done!